### PR TITLE
[PREVIEW] Add config for bulk-scan-orchestrator service

### PR DESCRIPTION
**Pull request by rkondratowicz**
Note that this new secret is read from the azure vault, not tactical hashicorp vault.
Added secrets in saat, aat and prod.

### CODE REVIEW

**Comment by timja** [comment]
how about demo too?

File: infrastructure/main.tf [hunk 1]
```diff
@@ -232,6 +233,7 @@ locals {
     MICROSERVICEKEYS_COH_COR                    = "${data.vault_generic_secret.cohcor.data["value"]}"
     MICROSERVICEKEYS_BULK_SCAN_PROCESSOR        = "${data.vault_generic_secret.bulkScanProcessor.data["value"]}"
     MICROSERVICEKEYS_BULK_SCAN_PROCESSOR_TESTS  = "${data.vault_generic_secret.bulkScanProcessorTests.data["value"]}"
+    MICROSERVICEKEYS_BULK_SCAN_ORCHESTRATOR     = "${local.microservice_key_names["BULK_SCAN_ORCHESTRATOR"]}"
```
Comment (lgonczar) [inline infrastructure/main.tf]: it has to be the secret here, so `local.microservice_key_settings[...`

File: infrastructure/main.tf [hunk 2]
```diff
@@ -187,6 +187,7 @@ locals {
     "COH_COR"                     = "microservicekey-coh-cor"
     "BULK_SCAN_PROCESSOR"         = "microservicekey-bulk-scan-processor"
     "BULK_SCAN_PROCESSOR_TESTS"   = "microservicekey-bulk-scan-processor-tests"
+    "BULK_SCAN_ORCHESTRATOR"      = "microservicekey-bulk-scan-orchestrator"
```
[the remark this answers is not in the document]
Comment (rkondratowicz) [reply infrastructure/main.tf]: done

File: infrastructure/main.tf (hunk 1 above)
Comment (rkondratowicz) [reply infrastructure/main.tf]: fixed

**Pull request merged**